### PR TITLE
Dept 73 dashboard

### PR DESCRIPTION
**Pull request by agonxalez**

### CODE REVIEW

**Review by blamboy090 — APPROVED** [review]
Excellent work! I did not see mobile styling as part of your AC so I am going to suggest we create a card in the backlog for that, otherwise LGTM!

File: components/dashboard/dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { DropdownProps } from 'ts/types';
+
+const Dropdown: React.FC<DropdownProps> = ({
+  options,
+  onChange,
+  value,
+  width,
+}) => {
+  const customStyles = {
+    option: (provided: any, state: { isSelected: any }) => ({
```
Comment (bdurel) [inline components/dashboard/dropdown.tsx:12]: Wondering why these have an any type? Is there no way to be more specific?

File: ts/types.ts [hunk 2]
```diff
@@ -45,6 +45,12 @@ export type FullProfile = BasicProfile & {
   shoutoutsReceived: Shoutout[];
   statusCode?: number;
 };
+export type DropdownProps = {
+  options: { value: string; label: string }[];
+  onChange: any;
```
Comment (bdurel) [inline ts/types.ts:50]: Wondering why theis has an any type? Is there no way to be more specific?

**Review by bdurel — CHANGES_REQUESTED** [review]
Hey Alberto, nice work. I've got some small questions to clear up in my comments regarding types and naming. I'm also wondering about Ben's mobile comment. I know that it doesn't mention mobile in the AC, but I was under the impression that mobile was implied unless otherwise stated. But maybe I'm incorrect here. Either way, it would be helpful to clear up. This might not be relevent if mobile/responsiveness is broken out into another card. It's also a very small thing, but on resize at around 50%, there's some bouncy behavior that I noticed when changing the year in the dropdown:
image
image

File: components/dashboard/index.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import Dropdown from '@/components/dashboard/dropdown';
 
 import styles from './dashboardcomp.module.scss';
```
Comment (bdurel) [inline components/dashboard/index.tsx:5]: I know this gets a bit confusing, but the components are now being named after the folder name. So this one should be dashboard.tsx as in components/dashboard/dashboard.tsx. Note: only using index for pages, so the index.tsx in pages/dashboard/index.tsx is correct as you currently have it.

File: components/dashboard/dashboardcomp.module.scss [hunk 4]
```diff
@@ -1,19 +1,19 @@
-.header {
```
Comment (bdurel) [inline components/dashboard/dashboardcomp.module.scss]: When you change components/dashboard/index.tsx to components/dashboard/dashboard.tsx, this file should be changed to dashboard.module.tsx.

**Review by bdurel — CHANGES_REQUESTED** [review]

**Review by bdurel — APPROVED** [review]

**Comment by bdurel** [comment]
LGTM!

**Pull request merged**